app/src/main/java/com/github/se/cyrcle/ui/list/ListScreen.kt

```diff
@@ -160,8 +216,15 @@ fun SpotListScreen(
       }
 }
 
+@SuppressLint("StateFlowValueCalledInComposition")
```

Comment:
I've been told suppressing this isn't always the right move. To quote @matttsss, "the annotation is here to help you". collectAsStates are your friend!

Reply:
I will look into it i think it is a superflux line that isn't necessary now.